src/ashell/jerry-code.c

```diff
@@ -356,28 +356,6 @@ int javascript_parse_code(const char *file_name, bool show_lines)
     buf = read_file_alloc(file_name, &size);
 
     if (buf && size > 0) {
-        if (show_lines) {
-            comms_printf("[READ] %d\n", (int)size);
-
-            // Print buffer test
-            int line = 0;
-            comms_println("[START]");
-            comms_printf("%5d  ", line++);
-            for (int t = 0; t < size; t++) {
-                uint8_t byte = buf[t];
-                if (byte == '\n' || byte == '\r') {
-                    comms_write_buf("\r\n", 2);
-                    comms_printf("%5d  ", line++);
-                } else {
-                    if (!isprint(byte)) {
-                        comms_printf("(%x)", byte);
-                    } else
-                        comms_writec(byte);
-                }
-            }
-            comms_println("[END]");
-        }
-
```

Comment:
Are you taking this out to save space? Seems kind of neat but I didn't know about it. You could give a 'v' arg on the command line when displaying code and get line numbers?